test/new_relic/agent/transaction_test.rb

```diff
@@ -1654,7 +1654,7 @@ def test_generates_guid_when_running_out_of_file_descriptors
         # descriptors
         # If the segment constructor fails to create a random guid, the
         # exception would be a RuntimeError
-        assert_raises Errno::EMFILE do
+        assert_raises Errno::EMFILE, Errno::ENFILE do
```

Comment:
Some tests fail with ubuntu latest that didn't fail on Ubuntu price (i.e. travis).  `Errno::ENFILE` gets returned instead of `Errno::EMFILE` -- both needed because macos will return `Errno::EMFILE`